bootstrap/gcp-terraform/gcp-terraform.tf

```diff
@@ -0,0 +1,90 @@
+# requires roles/resourcemanager.organizationViewer
+data "google_organization" "this" {
+  organization = local.organization_id
+}
+
+# The GCP account credentials used needs the following IAM permissions
+# - storage.buckets.create
+# - storage.buckets.list
+# - storage.objects.get
+# - storage.objects.create
+# - storage.objects.delete
+# - storage.objects.update
```

Comment:
This is only a subset of the permissions required. Best to just leave this out of here.